tests/PhpSpreadsheetTests/Reader/XlsxTest.php

```diff
@@ -1,19 +0,0 @@
-<?php
```

Comment:
This file and `without_cell_reference.xlsx` should probably not be deleted

Reply:
Sorry